src/control.c

```diff
@@ -31,7 +31,7 @@ xmp_context xmp_create_context(void)
 {
 	struct context_data *ctx;
 
-	ctx = calloc(1, sizeof(struct context_data));
+	ctx = (struct context_data*)calloc(1, sizeof(struct context_data));
```

Comment:
Minor style complaint, but libxmp pretty consistently spaces the left side of type `*`s from what I've seen (e.g. `struct context_data *`).

Reply:
Sorry about that - I should have checked what style the project uses.